automation/connect_to_rhel_console.exp

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/expect -f
 
+set timeout 300
+timeout {exit 1 }
+
 set kubeconfig [lindex $argv 0]
 set vm_name [lindex $argv 1]
+
 spawn ./virtctl --kubeconfig=$kubeconfig console $vm_name
-expect "Successfully connected to*\r"
-send -- "\r"
-expect "*login*\r"
+expect "*Welcome*\r"
```

Comment:
Reading the man page suggests:
expect timeout { exit 1 } "*Welcome*\r"

Reply:
Or using the default that should catch both timeout and EOF

`expect default { exit 1 } "Welcome\r"`